File: week04-2.py
# week04-2.py 是今天的leetcode 挑戰題 Easy 題(二合一)
# leetcode 2529. Maximum Count of Positive Integer and Negative Integer
#找出有幾個正數 有幾個負數
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
      pos = 0
      neg = 0
      for i in range(len(nums)):
          if nums[i] > 0:pos += 1
          if nums[i] < 0:neg += 1

      return max(pos, neg)

File: week04-2.c
// week04-2.c 琌さぱleetcode 珼驹肈 Easy 肈()
// leetcode 2529. Maximum Count of Positive Integer and Negative Integer
int maximumCount(int* nums, int numsSize) {

    int pos = 0, neg = 0; // 癹伴玡, 常琌0

    for(int i=0; i<numsSize; i++) {
        if( nums[i] > 0) pos++; //タ计
        if( nums[i] < 0) neg++; //璽计
    }


    if(pos>neg) return pos; // タ计ゑ耕, タ计
    else return neg; //ぃ礛,碞璽计
}
